lib/ansible/modules/cloud/openstack/os_image_facts.py

```diff
@@ -154,7 +172,7 @@ def main():
             module.exit_json(changed=False, ansible_facts=dict(
                 openstack_image=image))
         else:
-            images = cloud.list_images()
+            images = cloud.search_images(filters=module.params['properties'])
```

Comment:
I've not tested this, but does the filters have a default property of `None` or is it an empty set?

Reply:
The default value is None:
```python
    def search_images(self, name_or_id=None, filters=None):
        images = self.list_images()
        return _utils._filter_list(images, name_or_id, filters)
```
I've checked the _utils._filter_list returns the same list if filters is set to None.